### PR TITLE
Feature/order opening

### PR DESCRIPTION
**Pull request by daryush**
Simple initial implementation of automation for that feature.

Please run behat only for this feature by running `bin/behat features/order_opening.feature`
Feel free to run whole specs for project by running `bin/phpspec`

Feel invited for discussion.

### CODE REVIEW

File: features/bootstrap/OrderManagingContext.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    public function thereAreFollowingCaterers(TableNode $caterersParameters)
+    {
+        foreach ($caterersParameters as $catererParameters) {
+            $command = new Command\CreateNewCaterer($catererParameters['name']);
+            $this->commandBus->dispatch($command);
+        }
+    }
+
+    /**
+     * @When I start order for :catererName caterer
```
Comment (tabbi89) [inline features/bootstrap/OrderManagingContext.php:67]: I think we have to include information about who started an order, here UserId which could be random string would be enough for now also we need to distinguish orders so each Order should contain OrderId (uuid).

Comment (daryush) [inline features/bootstrap/OrderManagingContext.php:67]: It seems to be ok. In future that will be able to change that random user id, to some id given from some user provider or security component.

File: src/Eaty/Application/InMemoryOrders.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?php
```
Comment (daryush) [inline src/Eaty/Application/InMemoryOrders.php]: Move that class to Infrastructure/InMemory

File: src/Eaty/Application/InMemoryCaterers.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (daryush) [inline src/Eaty/Application/InMemoryCaterers.php]: Move that class to Infrastructure/InMemory

Comment (tabbi89) [inline src/Eaty/Application/InMemoryCaterers.php]: +1

Comment (daryush) [inline src/Eaty/Application/InMemoryCaterers.php]: Done

File: src/Eaty/Application/InMemoryOrders.php (hunk 2 above)
Comment (daryush) [inline src/Eaty/Application/InMemoryOrders.php]: Done

**Pull request merged**